README.md

```diff
@@ -0,0 +1,175 @@
+# Open Context for Node.js
+
+This is a simple library for providing Distributed Context Problem
+in Node.js programs.
+
+DCP is most beneficial in large service oriented architectures. Each request
+has some associated metadata, and we want to make sure that data stays with
+the tree of RPCs that service the root request.
+
+The `Context` object in this library stores that request metadata, or baggage.
+It's intended to be implemented into RPC libraries, who would be responsible
+for actually serializing the data onto the wire.
```

Comment:
RPC transport libraries are responsible for marshaling baggage on and off the wire.